packaging/conf/osinfo-defaults.properties

```diff
@@ -65,6 +65,7 @@ os.other.devices.display.protocols.value.4.3 = spice/qxl,vnc/vga,vnc/qxl
 os.other.devices.display.protocols.value.4.2 = spice/qxl,vnc/vga,vnc/qxl,vnc/cirrus
 os.other.devices.display.protocols.value.4.1 = spice/qxl,vnc/vga,vnc/qxl,vnc/cirrus
 os.other.devices.display.protocols.value.4.0 = spice/qxl,vnc/vga,vnc/qxl,vnc/cirrus
+os.other.devices.display.virtioVga.value = true
```

Comment:
For now, I'm not sure there will be no reason to disable it.

Reply:
so if it's just about the ability to disable it in case issues are found later on then a feature supported entry would probably be the better way to go

Reply:
Done.